### ISSUE TITLE
Module not loading if there is only one view

### ISSUE THREAD

**Issue by ludeeus** [maykar/compact-custom-header#54]
## Describe the bug
When you only have 1 view configured in Lovelace this module will not load.
Adding a "dummy" view is a workaround for the problem.

## Config

```yaml
resources:
  - url: /customcards/compact-custom-header.js
    type: module
....
      - clock_am_pm: true
        clock_format: '12'
        header: true
        main_config: true
        notifications: hide
        options: clock
        type: 'custom:compact-custom-header'
        voice: hide
```
_I use the UI to configure it, that's why this looks "wierd"_

## Error

_dev-console errors:_
```text
compact-custom-header.js:199 Uncaught (in promise) TypeError: Cannot read property 'querySelectorAll' of null
    at HTMLElement.run (compact-custom-header.js:199)
    at HTMLElement.buildConfig (compact-custom-header.js:168)
    at HTMLElement.updated (compact-custom-header.js:67)
    at HTMLElement.performUpdate (app-1017f5da.js:101)
    at HTMLElement._enqueueUpdate (app-1017f5da.js:101)
run @ compact-custom-header.js:199
buildConfig @ compact-custom-header.js:168
updated @ compact-custom-header.js:67
performUpdate @ app-1017f5da.js:101
_enqueueUpdate @ app-1017f5da.js:101
async function (async)
_enqueueUpdate @ app-1017f5da.js:101
requestUpdate @ app-1017f5da.js:101
set @ app-1017f5da.js:101
CompactCustomHeader @ compact-custom-header.js:50
(anonymous) @ VM356:1
create_card_element_createElement @ b932ff0e33612a653941.chunk.js:4741
createCardElement @ b932ff0e33612a653941.chunk.js:4741
createCardElement @ b932ff0e33612a653941.chunk.js:5594
_rebuildCard @ b932ff0e33612a653941.chunk.js:5666
element.addEventListener.ev @ b932ff0e33612a653941.chunk.js:5594
fireEvent @ app-1017f5da.js:1222
customElements.whenDefined.then @ b932ff0e33612a653941.chunk.js:4741
Promise.then (async)
createCardElement @ b932ff0e33612a653941.chunk.js:4741
createCardElement @ b932ff0e33612a653941.chunk.js:5594
config.cards.forEach @ b932ff0e33612a653941.chunk.js:5666
_createCards @ b932ff0e33612a653941.chunk.js:5666
updated @ b932ff0e33612a653941.chunk.js:5666
performUpdate @ app-1017f5da.js:101
_enqueueUpdate @ app-1017f5da.js:101
async function (async)
_enqueueUpdate @ app-1017f5da.js:101
requestUpdate @ app-1017f5da.js:101
set @ app-1017f5da.js:101
hui_view_HUIView @ b932ff0e33612a653941.chunk.js:5594
(anonymous) @ VM356:1
_selectView @ b932ff0e33612a653941.chunk.js:5888
async function (async)
_selectView @ b932ff0e33612a653941.chunk.js:5888
updated @ b932ff0e33612a653941.chunk.js:5888
performUpdate @ app-1017f5da.js:101
_enqueueUpdate @ app-1017f5da.js:101
async function (async)
_enqueueUpdate @ app-1017f5da.js:101
requestUpdate @ app-1017f5da.js:101
set @ app-1017f5da.js:101
hui_root_HUIRoot @ b932ff0e33612a653941.chunk.js:5666
_clone @ app-1017f5da.js:444
_commitTemplateResult @ app-1017f5da.js:215
commit @ app-1017f5da.js:215
render @ app-1017f5da.js:366
shady_render_render @ app-1017f5da.js:89
update @ app-1017f5da.js:135
performUpdate @ app-1017f5da.js:101
_enqueueUpdate @ app-1017f5da.js:101
async function (async)
_enqueueUpdate @ app-1017f5da.js:101
requestUpdate @ app-1017f5da.js:101
set @ app-1017f5da.js:101
_fetchConfig @ b932ff0e33612a653941.chunk.js:5924
async function (async)
_fetchConfig @ b932ff0e33612a653941.chunk.js:5924
firstUpdated @ b932ff0e33612a653941.chunk.js:5924
performUpdate @ app-1017f5da.js:101
_enqueueUpdate @ app-1017f5da.js:101
async function (async)
_enqueueUpdate @ app-1017f5da.js:101
requestUpdate @ app-1017f5da.js:101
set @ app-1017f5da.js:101
ha_panel_lovelace_LovelacePanel @ b932ff0e33612a653941.chunk.js:5888
(anonymous) @ VM356:1
loadingProm.then @ app-1017f5da.js:3075
Promise.then (async)
updated @ app-1017f5da.js:3075
performUpdate @ app-1017f5da.js:101
_enqueueUpdate @ app-1017f5da.js:101
async function (async)
_enqueueUpdate @ app-1017f5da.js:101
requestUpdate @ app-1017f5da.js:101
set @ app-1017f5da.js:101
commit @ app-1017f5da.js:215
commit @ app-1017f5da.js:215
update @ app-1017f5da.js:444
_commitTemplateResult @ app-1017f5da.js:215
commit @ app-1017f5da.js:215
render @ app-1017f5da.js:366
shady_render_render @ app-1017f5da.js:89
update @ app-1017f5da.js:135
performUpdate @ app-1017f5da.js:101
_enqueueUpdate @ app-1017f5da.js:101
async function (async)
_enqueueUpdate @ app-1017f5da.js:101
requestUpdate @ app-1017f5da.js:101
set @ app-1017f5da.js:101
commit @ app-1017f5da.js:215
commit @ app-1017f5da.js:215
update @ app-1017f5da.js:444
_commitTemplateResult @ app-1017f5da.js:215
commit @ app-1017f5da.js:215
update @ app-1017f5da.js:444
_commitTemplateResult @ app-1017f5da.js:215
commit @ app-1017f5da.js:215
render @ app-1017f5da.js:366
shady_render_render @ app-1017f5da.js:89
update @ app-1017f5da.js:135
performUpdate @ app-1017f5da.js:101
_enqueueUpdate @ app-1017f5da.js:101
async function (async)
_enqueueUpdate @ app-1017f5da.js:101
requestUpdate @ app-1017f5da.js:101
set @ app-1017f5da.js:101
_updateHass @ app-1017f5da.js:3139
services @ app-1017f5da.js:3139
n @ core-41c8125e.js:1
(anonymous) @ core-41c8125e.js:1
Promise.then (async)
c @ core-41c8125e.js:1
subscribe @ core-41c8125e.js:1
I @ core-41c8125e.js:1
window.hassConnection.then @ core-41c8125e.js:1
Promise.then (async)
151 @ core-41c8125e.js:1
__webpack_require__ @ core-41c8125e.js:1
(anonymous) @ core-41c8125e.js:1
(anonymous) @ core-41c8125e.js:1
```

_There was no errors in HA log_

## Screenshots

_One view:_
![image](https://user-images.githubusercontent.com/15093472/52970979-073ccb80-33b6-11e9-9d0b-71f15070ee27.png)

_More than one view:_
![image](https://user-images.githubusercontent.com/15093472/52971018-289db780-33b6-11e9-8742-b23bf9b95a20.png)

## Versions
Please include all of the following:
* Compact Custom Header Version: `1.0.0b5`
* Home Assistant version: `0.88.0b2`
* YAML or storage mode: both
* Browser type and version: Chrome (72.0.3626.109)

## Additional context

_Not sure what more to put here 🤷‍♂️_

![](https://media.giphy.com/media/dM8fBhhhAmbaU/giphy.gif)

**Comment by maykar** [comment]
On it, fix coming in next version. Thank you!

**Comment by maykar** [comment]
Fixed in 1.0.0b6
